cookiecutter/hooks.py

```diff
@@ -79,11 +79,21 @@ def run_script(script_path, cwd='.'):
     utils.make_executable(script_path)
 
     try:
-        proc = subprocess.Popen(script_command, shell=run_thru_shell, cwd=cwd)  # nosec
-        exit_status = proc.wait()
+        proc = subprocess.Popen(
+            script_command,
+            shell=run_thru_shell,  # nosec
+            cwd=cwd,
+            stderr=subprocess.PIPE,  # pipe only errors
+            universal_newlines=True,  # convert from b' format
+        )
+        # catch error during hook execution, if any
+        sp_output, sp_error = proc.communicate()
```

Comment:
`sp_output` is never used, so I recommend using an underscore as the variable name, which is common when indicating that the variable is unused.

```suggestion
        _, sp_error = proc.communicate()
```